Resources/Prototypes/SimpleStation14/Loadouts/jobs.yml

```diff
@@ -205,6 +213,7 @@
   exclusive: true
   jobWhitelist:
     - MedicalDoctor
+    - ChiefMedicalOfficer
```

Comment:
- I feel as though this is unnecessary. The CMO is a Head. Unlike a standard MD, they are not a surgeon, or chemist, or nurse, or otherwise.

```suggestion
```